Core/AppUrls.swift

```diff
@@ -114,11 +126,14 @@ public struct AppUrls {
     }
 
     public func applyStatsParams(for url: URL) -> URL {
-        let searchUrl = url.addParam(name: Param.source, value: ParamValue.source)
-            .addParam(name: Param.appVersion, value: appVersion)
+        var searchUrl = url.addParam(name: Param.source, value: ParamValue.source)
+        searchUrl = searchUrl.addParam(name: Param.appVersion, value: appVersion)
```

Comment:
I prefer the chained approach myself, it's designed so that you can assign to a let in a single statement... not fussed though.

Reply:
Me too! I just switched for consistency within the method as there's a spot where we can't chain. If you're not fussed, I'll keep it. Last chance to speak now if you want it changed...

Reply:
Me too! I just switched for consistency within the method as there's a spot where we can't chain. If you're not fussed, I'll keep it. Last chance to speak now if you want it changed...